auth/server/base/src/main/java/org/wildfly/security/auth/server/RequestInformationCallback.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.security.auth.server;
+
+import org.wildfly.security.auth.callback.ExtendedCallback;
+
+import java.net.URI;
+
+/**
+ * A {@link javax.security.auth.callback.Callback} to inform a server authentication context about current authenticaiton request.
+ *
+ */
+public class RequestInformationCallback implements ExtendedCallback {
+
+    /**
+     * request URI
+     */
+    private final URI requestUri;
+
+    public  RequestInformationCallback(URI requestUri) {
+        this.requestUri = requestUri;
+    }
+
+    /**
```

Comment:
Just minor but since this is public API, we should also add a description for the method.

Reply:
Fixed